dev/src/bulk-writer.ts

```diff
@@ -502,12 +504,16 @@ export class BulkWriter {
    * });
    */
   close(): Promise<void> {
+    if (!this.closed) {
```

Comment:
```suggestion
    this.verifyNotClosed();
    this.firestore.decrementBulkWritersCount();
```

This makes it more obvious since this.flush() calls it as well, but in a non-obvious way.

Reply:
this makes more sense, thanks